demos/README.md

```diff
@@ -0,0 +1,45 @@
+![buildah logo](https://cdn.rawgit.com/projectatomic/buildah/master/logos/buildah-logo_large.png)
+
+# Useful Buildah links
```

Comment:
Shoot, you got this backwards.  I wanted you to add a link on the main Buildah/README.md to add a pointer to this readme...

**[Demos](../demos/README.md)**

Reply:
I'd not add these links here, just keep them in the main.  Sorry for the tsweeney-confusion path I sent you on.